src/main/java/models/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package models;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANGE = 100;
+    private ColorSupplier colorSupplier;
+    private int ballNumber;
+
+    public Ball getRandomBall() {
+        this.colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        this.ballNumber = new Random().nextInt(RANGE);
```

Comment:
random also should be class field
```suggestion
        this.ballNumber = random.nextInt(RANGE);
```

Reply:
++